src/main/java/org/schemaspy/service/TableService.java

```diff
@@ -572,6 +572,13 @@ private void initIndexes(Database db, Table table) throws SQLException {
         }
     }
 
+    //This is to handle a problem with informix and lvarchar Issue 215. It's been reported to IBM.
+    //According to DatabaseMetaData.getIndexInfo() ORDINAL_POSITION is zero when type is tableIndexStatistic.
+    //Problem with informix is that lvarchar is reported back as TYPE = tableIndexOther and ORDINAL_POSITION = 0.
+    private boolean isIndexRow(ResultSet rs) throws SQLException {
+        return rs.getShort("TYPE") != DatabaseMetaData.tableIndexStatistic && rs.getShort("ORDINAL_POSITION") > 0;
```

Comment:
If type is tableIndexStatics it will by the negation become false, and isIndexRow will return false.
If type is other than tableIndexStatics it will evaluate true and the second condition will be checked.

> ORDINAL_POSITION short => column sequence number within index; zero when TYPE is tableIndexStatic

Zero is not within index so in reality the type check could be dropped and have only check ORDINAL_POSITION > 0.

But then again, a ticket has been submitted to IBM and Informix development.
And then again I don't think we should assume all JDBC drivers are implemented according to specification all of the time.